### PR TITLE
Add sondehub

### PR DESCRIPTION
**Pull request by TheSkorm**
*Description of changes:*

Add's sondehub radiosonde telemetry data S3 bucket.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 98ec49e78738fb63299b23c5f2303dc919490dbb
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e3b3a36a-53c7-4093-87ab-980ffd4bbb2b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/sondehub-telemetry.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - weather
+  - GPS
+License: Creative Commons BY-SA 2.0
+Resources:
+  - Description: Radiosonde Telemetry as JSON blobs of Universal Telemetry format prefixed by date/time
+    ARN: arn:aws:s3:::sondehub-open-data
+    Region: us-east-1
+    Type: S3 Bucket
+    Explore: http://sondehub-open-data.s3-website-us-east-1.amazonaws.com/#serial/
+  - Description: Radiosonde Telemetry as JSON blobs of Universal Telemetry format prefixed by serial number
```
Comment (zflamig) [inline datasets/sondehub-telemetry.yaml:25]: Lets just list the bucket once and then Explore should be an array of links so you can have By Date/time & Serial Number options there

File: datasets/sondehub-telemetry.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  data for an average of 384 radiosondes a day.
+
+  Data contains telemetry frames in Universal Sonde Telemetry format, including gps position, type,
+  altitude, pressure, temperature and humidity.
+Documentation: https://github.com/projecthorus/sondehub-infra/wiki/Amazon-Open-Data
+Contact: Michaela Wheeler <vk3fur@sondehub.org>
+ManagedBy: SondeHub
+UpdateFrequency: Data is updated as we receive it
+Tags:
+  - climate
```
Comment (zflamig) [inline datasets/sondehub-telemetry.yaml:14]: please add `aws-pds` as a tag. We use this to indicate participation in the Open Data Sponsorship Program

File: datasets/sondehub-telemetry.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Description: >
+  SondeHub Radiosonde telemetry contains global radiosonde (weather balloon) data captured by SondeHub from our
+  participating radiosonde_auto_rx receiving stations. Currently 313 receiver stations are providing
+  data for an average of 384 radiosondes a day.
+
+  Data contains telemetry frames in Universal Sonde Telemetry format, including gps position, type,
+  altitude, pressure, temperature and humidity.
+Documentation: https://github.com/projecthorus/sondehub-infra/wiki/Amazon-Open-Data
+Contact: Michaela Wheeler <vk3fur@sondehub.org>
+ManagedBy: SondeHub
```
Comment (zflamig) [inline datasets/sondehub-telemetry.yaml:11]: Do you have a logo you would want to appear here too? You can email me it if so

Comment (TheSkorm) [reply datasets/sondehub-telemetry.yaml:11]: That's a good question! We don't have a logo specifically for sondehub yet but it's probably something we should work on for our new website. We usually use the radiosonde_auto_rx logo but I don't think it makes sense here so it's probably easier if we leave it blank for the moment.

**Comment by TheSkorm** [comment]
Updated with suggestions :)

**Comment by zflamig** [comment]
Thanks!

**Pull request merged**